src/main/java/fallingleaves/fallingleaves/mixin/LeafTickMixin.java

```diff
@@ -24,6 +26,7 @@
 import java.util.List;
 import java.util.Random;
 
+@Environment(EnvType.CLIENT)
 @Mixin(LeavesBlock.class)
 public class LeafTickMixin {
```

Comment:
It's good practice to make mixins abstract

Reply:
Had no idea, what's the reasoning behind this?